src/os/unix/pa_pthread_util.c

```diff
@@ -83,7 +85,13 @@ PaUtilClockId PaPthreadUtil_NegotiateCondAttrClock( pthread_condattr_t *cattr )
 void PaPthreadUtil_GetTime( PaUtilClockId clockId, struct timespec *ts )
 {
 #if PAUTIL_USE_POSIX_ADVANCED_REALTIME
-    clock_gettime(clockId, ts);
+    if ( clock_gettime(clockId, ts) != 0 )
+    {
+        ts->tv_sec = 0;
+        ts->tv_nsec = 0;
```

Comment:
The most likely reason for clock_gettime failing is that the kernel detected that writing to `ts` would cause SIGSEGV. I'd wrap these assignments in `if (errno != EFAULT) {}` or move them below PA_DEBUG.

Reply:
Never mind. I see you already did the latter.